src/UI/UIView.cpp

```diff
@@ -1248,9 +1301,73 @@ void UIView::load_user_session(std::string filePath)
       {
         load_trans_functions(loadFile);
       }
+      else if (tag == "POI")
+      {
+        load_camera_poi(loadFile, std::stoi(vals[1]));
+      }
```

Comment:
This can be done for all the single line statements
```suggestion
      else if(tag == "POI") load_camera_poi(loadFile, std::stoi(vals[1]));
```

Reply:
With this many else `if` statements this should really be a `switch` block